code/modules/research/xenobiology/crossbreeding/warping.dm

```diff
@@ -0,0 +1,180 @@
+	effect_desc = "Forms a recoverable rune that lowers the body temperature of creatures that enter its circle."
+
+/obj/item/slimecross/warping/silver
+	colour = "silver"
+	rune_path = /obj/effect/slimerune/silver
+	effect_desc = "Forms a recoverable rune that absorbs food to feed those who cross its path."
+	var/nutrition = 0
+/*
+/obj/item/slimecross/warping/bluespace //TODO: Fix issues.
+	colour = "bluespace"
+	rune_path = /obj/effect/slimerune//bluespace
+	effect_desc = "Forms a recoverable rune that links a <b>wormhole satchel</b> with the space above it."
+*/
+/obj/item/slimecross/warping/sepia
+	colour = "sepia"
+	rune_path = /obj/effect/slimerune/sepia
+	effect_desc = "Forms a recoverable rune that holds any living creature in its influence under stasis."
+
+/obj/item/slimecross/warping/cerulean
+	colour = "cerulean"
+	rune_path = /obj/effect/slimerune/cerulean
+	effect_desc = "Forms a recoverable rune that displays a hologram of the last creature to cross it."
+
+/obj/item/slimecross/warping/pyrite
+	colour = "pyrite"
+	rune_path = /obj/effect/slimerune/pyrite
+	effect_desc = "Forms a recoverable rune that paints things that cross its area a rainbow of colors."
+
+/obj/item/slimecross/warping/red
+	colour = "red"
+	rune_path = /obj/effect/slimerune/red
+	effect_desc = "Forms a recoverable rune that increases the effectiveness of unarmed attacks made from within it."
+
+/obj/item/slimecross/warping/green
+	colour = "green"
+	rune_path = /obj/effect/slimerune/green
+	effect_desc = "Forms a recoverable rune that converts sheets of plasma to resin, allowing you to form xenomorphic structures."
+/*
+/obj/item/slimecross/warping/pink
+	colour = "pink"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that makes hugging anyone standing in it a pleasant experience."
+
+/obj/item/slimecross/warping/gold
+	colour = "gold"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that accepts coinage for items drawn from an unknown dimension."
+
+/obj/item/slimecross/warping/oil
+	colour = "oil"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that absorbs explosions, converting them into a gooey energy."
+
+/obj/item/slimecross/warping/black
+	colour = "black"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that summons the spirits of the imminently deceased."
+
+/obj/item/slimecross/warping/lightpink
+	colour = "light pink"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that stops simple creatures from crossing its boundary."
+
+/obj/item/slimecross/warping/adamantine
+	colour = "adamantine"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that crystallizes materials based on nearby ore nodes."
+
+/obj/item/slimecross/warping/rainbow
+	colour = "rainbow"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that opens a gate to a pocket dimension."
+*/
```

Comment:
Don't leave in commented out code like this

Reply:
I was intending to complete these in a later PR, but thinking about it, simply commenting these out is a really ineffective way of conveying that compared to _simply making them in a new PR._ Will do.